docs/installation/performance_optimizations.asciidoc

```diff
@@ -18,6 +18,101 @@ endif::[]
+
+* Enable NT Key cache.
+* Specify a cache expiration time. Default is 3600 (in seconds)
+* Fill in the Windows Group Policy Settings
+ ** Account Lockout Threshold
+ ** Reset Account Lockout Counter After
+ ** Account Lockout Duration
+ ** Old Password Allowed Period
+
+Those settings can be found on the Windows Domain Controller by the following steps:
+
+Go to _Start menu -> Administrative Tools -> Group Policy Management_
+In the console tree, expand *Forest* -> *Domains* -> *Your Domain* -> *Group Policy Objects* -> *Default Domain Policy*
+In the right panel, navigate to *Settings* page, You will have these parameters in *Policies* -> *Windows Settings* -> *Security Settings* -> *Account Policies, Account lockout policies*
+image::domain-windows-account-policy.png[scaledwidth="100%",alt="Windows Account Policies"]
+
+NOTE: Steps may vary on different windows versions. The steps above are for Windows Server 2022.
+
+NOTE: If you didn't see some of the values listed above, means they are set to default value. You can leave the settings in blank or as-is in PacketFence's configuration.
+
+NOTE: If you'd like to change the values, follow the default Windows GPO best practise guide.
+
+For *Old Password Allowed Period*, There's currently no GPO settings. The default value is 60 min. It can be changed following this guide:
+
+https://learn.microsoft.com/en-US/troubleshoot/windows-server/windows-security/new-setting-modifies-ntlm-network-authentication
+
+You'll still need to create a dedicated user that receives Windows Events from Domain Controller. You can create the user by the following steps:
+* log on to PacketFence Admin UI, in *Users* section, create a user with a strong, unique password.
+* change the user's *Access Level* to *Windows Event Receiver NTLM*
+
+This username and password will be required when you set up the Windows Event Notifier PowerShell script later.
+
+==== Active Directory Configuration
+
+Once configuration is done on PacketFence side, you'll still need some changes made on Windows Domain Controller
+
+===== Config Windows Event Notifier
+
+Windows Event Notifier is a powershell script used to filter, analyze and report accout events to PacketFence.
+Includes:
+ * Password reset
+ * Password change
+ * User account locked
+ * User account unlocked
+ * User account disabled
+
+
+
+*Password reset* and *Password change* Events must be reported to ensure Windows NT Key cache works properly.
+Other events listed above will be optional but recommended.
+
+To attach the Windows Event Notifier, you'll need
+
+ * copy the windows event notifier powershell script in /usr/local/pf/addons/AD/password_change_notifier.ps1 to *each* domain controller you want to have NT Key cache enabled.
+ * Modify the script, Fill in the PacketFence server IP and username and password of Event Reporter.
+ * Go to windows event viewer, try to find an event of specific category. e.g. 4724 for password reset and 4723 for password change.
```

Comment:
IMO we should explain how to configure it via the "Task Scheduler" instead

Reply:
Done. added detail steps and images as well.